### PR TITLE
Initial commit to datamodel media contribution.

### PR DESCRIPTION
**Pull request by RanjithTom**
Currently the context only have 2 enums such as MediaSubType and MediaFormat. Need to add more fields and publication instances in coming commits

### CODE REVIEW

File: nva-datamodel-java/src/main/java/no/unit/nva/model/contexttypes/media/MediaFormat.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum MediaFormat {
+    TEXT("Text"),
+    SOUND("Sound"),
+    VIDEO("Video");
+
+    private String value;
+
+    MediaFormat(String value) {
```
Comment (axthosarouris) [inline nva-datamodel-java/src/main/java/no/unit/nva/model/contexttypes/media/MediaFormat.java:12]: it needs a @JsonCreator method . You should look how we implement other enums

File: nva-datamodel-java/src/main/java/no/unit/nva/model/contexttypes/media/MediaSubType.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+    JOURNAL("Journal"),
+    RADIO("Radio"),
+    TV("TV"),
+    INTERNET("Internet"),
+    OTHER("Other");
+
+    private String value;
+
+    MediaSubType(String value) {
```
Comment (axthosarouris) [inline nva-datamodel-java/src/main/java/no/unit/nva/model/contexttypes/media/MediaSubType.java:15]: ditto

File: nva-datamodel-java/src/main/java/no/unit/nva/model/instancetypes/media/MediaBase.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonSetter;
+import java.util.Objects;
+import no.unit.nva.model.instancetypes.PublicationInstance;
+import no.unit.nva.model.pages.NullPages;
+
+public class MediaBase implements PublicationInstance<NullPages> {
+
+    @JsonProperty("pages")
+    public static final NullPages pages = new NullPages();
```
Comment (axthosarouris) [inline nva-datamodel-java/src/main/java/no/unit/nva/model/instancetypes/media/MediaBase.java:14]: static values are capital, name it NULL_PAGES

**Review by axthosarouris — CHANGES_REQUESTED** [review]
In future PRs, do not commit the json files because the PR is very heavy for the browser. Make the changes in the model, and then in another PR update the version and the json fles.

**Review by brinxmat — COMMENTED** [review]
See other reviewer's comments

**Pull request merged**